src/setupSessionId.ts

```diff
@@ -0,0 +1,25 @@
+import { ulid } from 'ulidx';
+
+export const SESSION_ID_KEY = 'yext_analytics_session_id';
```

Comment:
This entire file is copied from how chat (or the previous iteration of the sdk) was getting sessionId.